mbtools/validate_mbz_html.py

```diff
@@ -11,13 +11,20 @@
 HREF_VIOLATION = "ERROR: Uses invalid 'href' value in <a> tag"
 
 VALID_PREFIXES = ["https://s3.amazonaws.com/im-ims-export/",
+                  "https://osx-int-alg.s3",
+                  "https://openstax.org/apps/archive",
                   "https://k12.openstax.org/contents/raise"]
```

Comment:
@rnathuji do all of these openstax prefixes look reasonable?

Reply:
Oh, we should not include these as they are valid failures. The first is a reference to their S3 bucket (which we don't want) and the latter to the books S3 bucket which if they use the content we've provided should always use `https://k12.openstax.org/contents/raise`.

WW did mention today they have not yet made the fixes to incorporate that content, so we can use these failures to confirm that they do.

Reply:
@rnathuji I'm not yet clear on this. Of these four, we should only mark the last one as valid? Or none?

Reply:
The original list with `"https://s3.amazonaws.com/im-ims-export/"` and `"https://k12.openstax.org/contents/raise"` and should be good for this one. The former we'll eventually address in [k12-62](https://github.com/openstax/k12/issues/62) as a transform step, but we need to allow it from WW as those were included in the original IM content.